src/styles/card_styles.scss

```diff
@@ -0,0 +1,99 @@
+/* #region card */
```

Comment:
You have a separate file with styles for one element, this comment is not needed here.

```suggestion
```